From f7fc9d9b0f167b4d6520042370f6c145970295b9 Mon Sep 17 00:00:00 2001
From: Gustavo Duarte <gduarte@redhat.com>
Date: Mon, 1 Sep 2014 20:15:29 -0400
Subject: [powerpc] perf/hv_24x7: Probe errors changed to pr_debug(), padding fixed

Message-id: <1409602534-23951-11-git-send-email-gduarte@redhat.com>
Patchwork-id: 90120
O-Subject: [RHEL7.1 PATCH BZ 947159 10/15] powerpc/perf/hv_24x7: Probe errors changed to pr_debug(), padding fixed
Bugzilla: 947159
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=947159
Upstream Status: e98bf005d5feafb3a089868420c1f3743d304e14

commit e98bf005d5feafb3a089868420c1f3743d304e14
Author: Cody P Schafer <cody@linux.vnet.ibm.com>
Date:   Tue Apr 15 10:10:50 2014 -0700

    powerpc/perf/hv_24x7: Probe errors changed to pr_debug(), padding fixed

    fixup for "powerpc/perf: Add support for the hv 24x7 interface"

    Makes the "not enabled" message less awful (and hides it in most cases).

    Signed-off-by: Cody P Schafer <cody@linux.vnet.ibm.com>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/perf/hv-24x7.c b/arch/powerpc/perf/hv-24x7.c
index 297c910..f5bca73 100644
--- a/arch/powerpc/perf/hv-24x7.c
+++ b/arch/powerpc/perf/hv-24x7.c
@@ -485,13 +485,13 @@ static int hv_24x7_init(void)
  struct hv_perf_caps caps;
 
  if (!firmware_has_feature(FW_FEATURE_LPAR)) {
-  pr_info("not a virtualized system, not enabling\n");
+  pr_debug("not a virtualized system, not enabling\n");
   return -ENODEV;
  }
 
  hret = hv_perf_caps_get(&caps);
  if (hret) {
-  pr_info("could not obtain capabilities, error 0x%80lx, not enabling\n",
+  pr_debug("could not obtain capabilities, not enabling, rc=%ld\n",
     hret);
   return -ENODEV;
  }
-- 
1.7.1